OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnHealth.cs

```diff
@@ -0,0 +1,100 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Applies a condition to the actor at when its health is between 2 specific values.")]
```

Comment:
The wording on this could be a bit better.

For example:

```
Applies a condition to this actor when its health is between 2 values
```

Also see my comment on inclusive vs exclusive range which would change the wording of this.

Either way this trait's docs should mention whether the values are inclusive or exclusive.

Reply:
Those trait descs currently aren't really visible anywhere outside the code itself, and the descs of MinHP/MaxHP implicitly mention that they're inclusive.

Reply:
The trait descs are directly visible on the trait wiki page.

You can't "implicitly mention" something...

Reply:
> The trait descs are directly visible on the trait wiki page.

Ok, I guess I missed that.

> You can't "implicitly mention" something...

I meant he didn't explicitly use 'inclusive' or 'includes', but "Maximum level (...) at which (...)", but I see your point.